widget/class-widget.php

```diff
@@ -31,7 +31,10 @@ function __construct() {
    * @param array $args     Widget arguments.
    * @param array $instance Saved values from database.
    */
-   public function widget( $args, $instance ) {
```

Comment:
I can't comment on opps-list-widget.php within this commit, so I'll add a note here. I noticed that you went with <li> items for each opportunity title. I like it from an HTML perspective, but it adds bullets to each title and actually makes the "When:" text stand out more than the title since it's bold. I know you were going to talk to Alex about this, so I'd love to hear what led to this approach.